### DISCUSSION TITLE
Dictionary

### DISCUSSION THREAD

**Discussion by lagleki** [lojban/casnu#8] [General]
Might be considered a part of [The CLL](https://github.com/lojban/casnu/discussions/6) project
- good non-cryptic definitions of all core words including cmavo. Not conflicting with definitions in the CLL
- correct usage examples similar or better than in Loglan1
- illustrations

**Comment by lagleki** [comment]
Picture dictionary: https://docs.google.com/spreadsheets/d/1Md0pojdcO3EVf3LQPHXFB7uOThNvTWszkWd5T4YhvKs/edit#gid=1254040829

**Reply by ghost** [reply]
Holy cow
